front/app/modules/commercial/moderation/admin/containers/index.tsx

```diff
@@ -462,6 +454,11 @@ const Moderation = memo<Props & InjectedIntlProps>(({ className, intl }) => {
                 <Outlet
                   id="app.modules.commercial.moderation.admin.containers.tabs"
                   onData={handleData}
+                  activeFlagsCount={
```

Comment:
You can ignore this one, it'll be replaced by a value generated by the backend later